livre/core/render/Frustum.h

```diff
@@ -36,7 +36,15 @@ namespace livre
 class Frustum : public Frustumf
 {
 public:
-    LIVRECORE_API Frustum();
+
+    /**
+     * Initializes the Frustum parameters from the modelview and projection matrices.
+     * @param modelViewMatrix 4x4 modelview matrix.
+     * @param projectionMatrix 4x4 projection matrix,
+     */
+    LIVRECORE_API Frustum( const Matrix4f& modelViewMatrix = Matrix4f::IDENTITY,
+                           const Matrix4f& projectionMatrix = Matrix4f::IDENTITY );
```

Comment:
I find default parameters fishy. Can't you simply call Frustum( Matrix4f(), Matrix4f()) in the places where needed? IDENTITY in any case will disappear soon since the default ctor does the same.